packages/graphql/tests/integration/subscriptions/spatial-types.int.test.ts

```diff
@@ -0,0 +1,729 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { faker } from "@faker-js/faker";
+import { gql } from "apollo-server";
+import { graphql } from "graphql";
+import { Driver, int, Session } from "neo4j-driver";
+import { Neo4jGraphQL } from "../../../src";
+import { cleanNodes } from "../../utils/clean-nodes";
+import { generateUniqueType } from "../../utils/graphql-types";
+import { TestSubscriptionsPlugin } from "../../utils/TestSubscriptionPlugin";
+import Neo4j from "../neo4j";
+
+describe("Subscriptions to spatial types", () => {
+    let driver: Driver;
+    let neo4j: Neo4j;
+    let session: Session;
+    let neoSchema: Neo4jGraphQL;
+    let plugin: TestSubscriptionsPlugin;
+
+    const typeMovie = generateUniqueType("Movie");
+
+    beforeEach(async () => {
+        neo4j = new Neo4j();
+        driver = await neo4j.getDriver();
+        plugin = new TestSubscriptionsPlugin();
+        const typeDefs = gql`
+            type ${typeMovie} {
+                title: String!
+                synopsis: String
+                filmedIn: Point
+                location: CartesianPoint
+            }
+        `;
+
+        neoSchema = new Neo4jGraphQL({
+            typeDefs,
+            config: { enableRegex: true },
+            plugins: {
+                subscriptions: plugin,
+            },
+        });
+        session = await neo4j.getSession();
+    });
+
+    afterEach(async () => {
+        await cleanNodes(session, [typeMovie]);
+        await session.close();
+        await driver.close();
+    });
+
+    test("create type with Point field", async () => {
```

Comment:
I see these are very similar tests, just done to Point and cartesian points. Would it be possible to reuse the same tests and just change the input with `test.each`?
 
 I assume is not that simple, so I'm only mentioning it in case it is relatively easy to do

Reply:
Not much else is similar, they are different fields in the graphql type and even if they weren't, the input types (PointInput, CartesianPointInput) have different fields (long/lat/height vs x/y/z). It can be done but so much has to be specified that I don't see the benefit in uniting them, unless you have a strong opinion on this :D